doc/source/examples/unresolved-cfd-dem/dense-pneumatic-conveying/dense-pneumatic-conveying.rst

```diff
@@ -0,0 +1,636 @@
+     set enable               = true
+     set enable beta particle = true
+     set average velocity     = 3
+     set flow direction       = 0
+     set beta threshold       = 0.05
+     set alpha                = 0.25
+     set verbosity            = verbose
+   end
+
+Void Fraction
+~~~~~~~~~~~~~~~
+
+We choose the `quadrature centred method (QCM) <../../../theory/multiphase/cfd_dem/unresolved_cfd-dem.html#the-quadrature-centered-method>`_  to calculate the void fraction. The ``l2 smoothing factor`` we choose is the square of twice the diameter of the particles.
+
+.. code-block:: text
+
+    subsection void fraction
+      set mode                = qcm
+      set read dem            = true
+      set dem file name       = dem
+      set l2 smoothing factor = 0.0001
+    end
+
+CFD-DEM
+~~~~~~~~~~
+
+The chosen drag model is Di Felice, and we use the Saffman lift force, the buoyancy force, and the pressure force. The coupling frequency is set to 100, which means that the DEM time step is 5e-6 s, for a Rayleigh critical time step of about 3.5 %. The grad-div stabilization is used with a length scale of 0.084, the diameter of the pipe.
+
+.. code-block:: text
+
+   subsection cfd-dem
+     set grad div               = true
+     set drag model             = difelice
+     set saffman lift force     = true
+     set buoyancy force         = true
+     set pressure force         = true
+     set coupling frequency     = 100
+     set implicit stabilization = false
+     set grad-div length scale  = 0.084
+     set particle statistics    = true
+   end
+
+Non-linear Solver
+~~~~~~~~~~~~~~~~~
+
+We use the inexact Newton non-linear solver to minimize the number of time the matrix of the system is assembled. This is used to increase the speed of the simulation, since the matrix assembly requires significant computations.
+
+.. code-block:: text
+
+   subsection non-linear solver
+     subsection fluid dynamics
+       set solver           = inexact_newton
+       set matrix tolerance = 0.1
+       set reuse matrix     = true
+       set tolerance        = 1e-4
+       set max iterations   = 10
+       set verbosity        = quiet
+     end
+   end
+
+Linear Solver
+~~~~~~~~~~~~~
+
+.. code-block:: text
+
+   subsection linear solver
+     subsection fluid dynamics
+       set method                                = gmres
+       set max iters                             = 200
+       set relative residual                     = 1e-4
+       set minimum residual                      = 1e-6
+       set ilu preconditioner fill               = 0
+       set ilu preconditioner absolute tolerance = 1e-8
+       set ilu preconditioner relative tolerance = 1.00
+       set verbosity                             = quiet
+       set max krylov vectors                    = 200
+     end
+   end
+
+
+--------
+Results
+--------
+
+The results presented here are obtained from a custom post-processing code that is currently not provided with the example.
```

Comment:
Is there a reason why the raw python files are not just included in the example? Even if they are not fully clean or are slightly hardcoded, I would still include them so someone can re-run the things.

Reply:
I talk about it in the PR description. 
It is because I stopped using the current version Lethe PyVista a while ago since new changes made my code and my output incompatible with it. Also, I use custom classes for my plots and for the processing of logs and PyVista files that are not specifically related to this example. 
My codes need a clean up, I don't feel confortable to just dump them as is in an example honestly. 
I wanna do a real clean up of my functions and put them in Utilities, since I have nice stuff, but I need to make it compatible with current tools and test it.

Reply:
I understand, but then in the meantime I would remove the graphs from the results section then. The idea of the examples is that we include the tools necessary for the example to be re-run. If most of the graphs in the results cannot be remade from the example itself, then I don't think we should include them right now.
We can do this in a multiple stage way. Merge the example without the graphs that cannot be remade, and then once the utilities are put into place and the scripts can be made available, we add the results and the scripts.

Reply:
I can do a quick clean up for a standalone before the merge then. For me the fun part were the results, I perfer not to remove them :)

Reply:
I agree, the results analysis you add they add a lot to the example.